src/coreclr/tools/Common/Compiler/DependencyAnalysis/AssemblyStubNode.cs

```diff
@@ -68,6 +67,9 @@ public override ObjectData GetData(NodeFactory factory, bool relocsOnly)
                     arm64Emitter.Builder.AddSymbol(this);
                     return arm64Emitter.Builder.ToObjectData();
 
+                case TargetArchitecture.Wasm32:
+                    return new ObjectData(null, null, 0, null);
```

Comment:
Might be better to add a new virtual method for this. This is mostly so that the TentativeMethod node can override it to return `new ObjectData(null, new Relocation[] { the target goes here }...);`.